### PR TITLE
[UI] Enable form save button if logged out

### PR DESCRIPTION
**Pull request by aryamanpuri**
#1961

### CODE REVIEW

File: ui/src/components/WMWorkerPoolEditor/index.jsx [hunk 1]
```diff
@@ -286,6 +286,21 @@ export default class WMWorkerPoolEditor extends Component {
...
+      workerPoolId,
+    } = this.props.workerPool;
+    const isWorkerPoolDirty =
+      isNewWorkerPool ||
+      workerPool.description !== description ||
+      workerPool.emailOnError !== emailOnError ||
+      workerPool.owner !== owner ||
+      workerPool.providerId !== providerId ||
+      joinWorkerPoolId(workerPool.workerPoolId1, workerPool.workerPoolId2) !==
+        workerPoolId;
```
Comment (helfi92) [inline ui/src/components/WMWorkerPoolEditor/index.jsx]: STR:

1. Navigate to http://localhost:5080/worker-manager and make sure you are logged out
2. Click on any worker pool id that's under "Worker Pool ID"
3. Modify the code editor under "Configuration"

Expected: The save button becomes enabled.
Actual: Nothing happens.

Comment (aryamanpuri) [reply ui/src/components/WMWorkerPoolEditor/index.jsx]: My bad .Problem resolved , Have a look .
@helfi92

**Comment by aryamanpuri** [comment]
@helfi92 Any changes required?

**Comment by aryamanpuri** [comment]
> Thank you so much!

Can you assign me any task ?

**Pull request merged**